back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -65,7 +65,9 @@ def index_xlsx
       includes: %i[author topics project idea_status idea_files],
       paginate: false
     ).find_records
-
+    if current_user.project_moderator? && params[:project].blank?
```

Comment:
We don't usually call `project_moderator?` or `admin?` in controllers. Did you consider moving this logic somehow in the policy? And the same for the comments controller.

Reply:
thanks for explaining how to make that on the policy, now looks much simpler indeed!